core/run.go

```diff
@@ -121,13 +121,13 @@ func RateLimit(keyFn func(*http.Request) string, burst, perSecond int) RunOption
 // used for Chain Core Developer Edition to expose the configuration UI
 // in the dashboard. API authentication still applies to an unconfigured
 // Chain Core.
-func RunUnconfigured(ctx context.Context, db pg.DB, sdb *sinkdb.DB, routableAddress string, opts ...RunOption) *API {
+func RunUnconfigured(ctx context.Context, confOpts *config.Options, db pg.DB, sdb *sinkdb.DB, routableAddress string, opts ...RunOption) *API {
 	a := &API{
 		db:           db,
 		sdb:          sdb,
 		accessTokens: &accesstoken.CredentialStore{DB: db},
 		grants:       authz.NewStore(sdb, GrantPrefix),
-		options:      config.New(sdb),
+		options:      confOpts,
```

Comment:
Currently cmd/cored needs access to it because the block signer is initialized in cored. Eventually, I think we should try to push that initialization into chain/core, but it'll require a bit of refactoring around the mockhsm.